playbooks/azure.yml

```diff
@@ -0,0 +1,143 @@
+      "9":  "usgovvirginia"
+      "10":  "usgoviowa"
+      "11":  "usdodeast"
+      "12":  "usdodcentral"
+      "13":  "canadaeast"
+      "14":  "canadacentral"
+      "15":  "brazilsouth"
+      "16":  "southeastssia"
+      "17":  "eastssia"
+      "18":  "australiaeast"
+      "19":  "australiasoutheast"
+      "20":  "chinaeast"
+      "21":  "chinanorth"
+      "22":  "centralindia"
+      "23":  "westindia"
+      "24":  "southindia"
+      "25":  "japaneast"
+      "26":  "japanwest"
+      "27":  "koreacentral"
+      "28":  "koreasouth"
+      "29":  "northeurope"
+      "30":  "westeurope"
+      "31":  "germanycentral"
+      "32":  "germanynortheast"
+      "33":  "ukwest"
+      "35":  "uksouth"
+
+  # These variable files are included so the azure-security-group role
+  # knows which ports to open
+  vars_files:
+    - roles/openconnect/defaults/main.yml
+    - roles/openvpn/defaults/main.yml
+    - roles/shadowsocks/defaults/main.yml
+    - roles/ssh/defaults/main.yml
+    - roles/streisand-gateway/defaults/main.yml
+    - roles/stunnel/defaults/main.yml
+    - roles/tor-bridge/defaults/main.yml
+    - roles/wireguard/defaults/main.yml
+
+  vars_prompt:
+    - name: "azure_region_var"
+      prompt: >
+        What region should the server be located in?
+           1. East US             (Virginia)
+           2. East US 2           (Virginia)
+           3. Central US          (Iowa)
+           4. North Central US    (Illinois)
+           5. South Central US    (Texas)
+           6. West Central US     (West Central US)
+           7. West US             (California)
+           8. West US 2           (West US 2)
+           9. US Gov Virginia     (Virginia)
+          10. US Gov Iowa         (Iowa)
+          11. US DoD East         (US DoD East)
+          12. US DoD Central      (US DoD Central)
+          13. Canada East         (Quebec City)
+          14. Canada Central      (Toronto)
+          15. Brazil South        (Sao Paulo State)
+          16. Southeast Asia      (Singapore)
+          17. East Asia           (Hong Kong)
+          18. Australia East      (New South Wales)
+          19. Australia Southeast (Victoria)
+          20. China East          (Shanghai)
+          21. China North         (Beijing)
+          22. Central India       (Pune)
+          23. West India          (Mumbai)
+          24. South India         (Chennai)
+          25. Japan East          (Tokyo, Saitama)
+          26. Japan West          (Osaka)
+          27. Korea Central       (Seoul)
+          28. Korea South         (Busan)
+          29. North Europe        (Ireland)
+          30. West Europe         (Netherlands)
+          31. Germany Central     (Frankfurt)
+          32. Germany Northeast   (Magdeburg)
+          33. UK West             (Cardiff)
+          35. UK South            (London)
+        Please choose the number of your region. Press enter for default (#9) region.
+      default: "28"
```

Comment:
It's the closest one to my location. Do you a better suggestion for a default location ? :)